.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+inherit_gem:
+  rubocop-rootstrap:
+    - config/rails_edge.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
+AllCops:
+  Exclude:
+    - db/schema.rb
+    - bin/bundle
+    - bin/yarn
+  SuggestExtensions: false
+
+Layout/LineLength:
+  Exclude:
+    - config/environments/production.rb
```

Comment:
Can we fix this file and `bin/bundle` and `bin/yarn` instead of excluding them? I think it'd be an easy fix 🙏